src/main/java/wraith/alloyforgery/recipe/AlloyForgeRecipe.java

```diff
@@ -39,6 +37,7 @@ public class AlloyForgeRecipe implements Recipe<Inventory> {
     public static final Map<AlloyForgeRecipe, PendingRecipeData> PENDING_RECIPES = new HashMap<>();
 
     private final Identifier id;
+    private Optional<Identifier> secondaryID = Optional.empty();
```

Comment:
It would be nice to have a comment which explains what the purpose for this second identifier is